commitizen/providers/__init__.py

```diff
@@ -0,0 +1,28 @@
+from __future__ import annotations
+
+from typing import cast
+
+import importlib_metadata as metadata
+
+from commitizen.config.base_config import BaseConfig
+from commitizen.exceptions import VersionProviderUnknown
+
+from .base_provider import VersionProvider
```

Comment:
```suggestion
from commitizen.providers.base_provider import VersionProvider
```

Let's unify how we import modules